### DISCUSSION TITLE
Rules doesn't work

### DISCUSSION THREAD

**Discussion by moresol** [asmvik/yabai#2082] [Q&A] [answered]
Macbook air m1 sonoma latest. 
those apps are not float at all, even though, the rules are set, and scripting addition is updated.
I remember, a few weeks it was fine.
How do I fix it? 😖
![Screenshot 2024-02-07 at 3 41 29 AM](https://github.com/koekeishiya/yabai/assets/106124331/531de553-22bf-43b7-a61d-56aac1ac1c23)
![Screenshot 2024-02-07 at 4 17 13 PM](https://github.com/koekeishiya/yabai/assets/106124331/2bb7429f-737d-43a2-bbff-99abbd5272a7)

**Comment by asmvik** [comment, marked as answer]
Add `&` at the end of line 25, that starts borders.

**Reply by moresol** [reply]
I am so sorry, I'm blind! Saved me as always.
